File: String/1475.js
function solution(param) {
  let match = [];
  match.length = 10;
  match.fill(0, 0, 10);

  let strParam = param.toString();
  for (let e of strParam) {
    match[e] += 1;
  }

  match[6] += match[9];
  match[6] = Math.ceil(match[6] / 2);
  match[9] = 0;

  let ans = 0;
  match.forEach(e => {
    if (ans < e) {
      ans = e;
    }
  });
  console.log(ans);
}

solution(999966);
solution(123456789);
solution(112345678999);
solution(213213213);

File: String/1475.java
import java.util.Scanner;

public class Main {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);

		int[] alphbetHit = new int[10];

		char[] input = sc.nextLine().toCharArray();

		int len = input.length;

		for (int i = 0; i < len; i++) {
			int n = Character.getNumericValue(input[i]);

			if (n == 9) {
				alphbetHit[6] += 1;
			} else {
				alphbetHit[n] += 1;
			}
		}

		alphbetHit[6] = (int) Math.ceil(alphbetHit[6] / 2.0);
		int ans = Integer.MIN_VALUE;

		for (int i = 0; i < 10; i++) {
			if (ans < alphbetHit[i]) {
				ans = alphbetHit[i];
			}
		}
		System.out.println(ans);
	}
}
